# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsFrameLoaderOwner.cpp
# Commit: 95b350572bbf
# Full Hash: 95b350572bbf079fdf6613b5a721c6ee5023fc6e
# Author: Kashav Madan <kmadan@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1646563
# File Overlap Count: 1
# Description:
#   Bug 1646563 - Remove BrowserId from nsFrameLoaderOwner, r=nika
#   
#   We can just use BrowsingContext::BrowserId directly, so it's unnecessary to have
#   the field on nsFrameLoaderOwner as well.
#   
# ==============================================================================

diff -r 4eab38aba3fe -r 95b350572bbf dom/base/nsFrameLoaderOwner.cpp
--- a/dom/base/nsFrameLoaderOwner.cpp	Mon Jun 22 21:30:31 2020 +0000
+++ b/dom/base/nsFrameLoaderOwner.cpp	Mon Jun 22 21:46:03 2020 +0000
@@ -284,9 +284,3 @@
                          /* inProgress */ false,
                          /* isRemote */ false, frameLoaderInit, IgnoreErrors());
 }
-
-void nsFrameLoaderOwner::UnbindFromTree() {
-  // If we're being adopted into a different document, we'll want to inherit a
-  // browser ID from our new BrowsingContext, so clear our current ID here.
-  mBrowserId = 0;
-}